knowledge-base/radpdfprocessing-customize-textboxfield-colors.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Customizing Text and Border Colors to Highlight a TextBoxField with RadPdfProcessing
+description: Learn how to alter the text and border colors to highlight a TextBoxField within a PDF document using RadPdfProcessing.
+type: how-to
+page_title: How to Change Text and Border Colors to Highlight a TextBoxField with RadPdfProcessing
+slug: radpdfprocessing-customize-textboxfield-colors
+tags: pdfprocessing, document, processing, textbox, field, color, border, customization, pdf, highlight 
+res_type: kb
+ticketid: 1673638
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.4.1106| RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+When working with PDF documents containing [form fields]({%slug radpdfprocessing-model-interactive-forms-form-fields%}), you might need to update the text and border colors of a [TextBoxField]({%slug radpdfprocessing-model-interactive-forms-form-fields-textboxfield%}) and highlight specific fields. This article demonstrates how to customize the appearance of `TextBoxField` elements, including changing their text and border colors.
+
+![Highlight TextBox Field](images/highlight-textbox-field.gif)    
```

Comment:
The gif shows only changes to the background color as opposed to the text and border colors as stated in the description above.